src/NHibernate/Hql/Ast/ANTLR/Tree/FromElement.cs

```diff
@@ -544,11 +544,12 @@ public void HandlePropertyBeingDereferenced(IType propertySource, string propert
 						_dereferencedBySuperclassProperty = true;
 					}
 				}
-				catch (QueryException)
+				catch (QueryException ex)
 				{
 					// ignore it; the incoming property could not be found so we
 					// cannot be sure what to do here.  At the very least, the
 					// safest is to simply not apply any dereference toggling...
+					Log.Info(ex, "Unable to find property {0}, no dereference will be handled for it.", propertyName);
```

Comment:
This shall be at Debug level. Info is too high.

Reply:
This is for matching the [other log](https://github.com/nhibernate/nhibernate-core/pull/1555/files/9a4acd4d05683ed39a6b59398d8c27485c1dead0#diff-13423b9d5cda494b63c32e6413dd7f58R535). Not very important, so well:
 - change both to Debug?
 - just this one?
 - keep them both at Info?

Reply:
Just this one, thanks

Reply:
Done.